starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +116,16 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(arr,word){
+  return arr.reduce(function(acc,val){
```

Comment:
This is great, but maybe in this case would be more interesting using a for that you can stop as soon as you encounter the value. That way, if the word is at the beginning of the array, you don't iterate over every element.